src/ui/input/input_wrap.jsx

```diff
@@ -40,8 +40,8 @@ export default class InputWrap extends React.Component {
         <div className={wrapClassName}>
           {iconElement}
           {this.props.children}
+          {after}
```

Comment:
The component `after` is only ever the password strength field. I've made it positioned relative to this div, because it doesn't change height (the outer div does depending on the value of `errorTooltip`)